linked_list.h

```diff
@@ -38,6 +39,20 @@ void print_float(void * a) {
 	printf("%0.2f ", *(float *)a);
 }
 
+size_t get_size(char type, void *data) {

```

Comment:
Dont use this, no type -> function mapping.

Reply:
Didnt get it.